api/src/services/email.service.ts

```diff
@@ -1,34 +1,31 @@
+import * as aws from '@aws-sdk/client-sesv2';
+import { HttpService } from '@nestjs/axios';
 import { Inject, Injectable, Logger } from '@nestjs/common';
 import { ConfigService } from '@nestjs/config';
-import { HttpService } from '@nestjs/axios';
+import {
+  LanguagesEnum,
+  ListingEventsTypeEnum,
+  ReviewOrderTypeEnum,
+} from '@prisma/client';
+import dayjs from 'dayjs';
+import advanced from 'dayjs/plugin/advancedFormat';
+import tz from 'dayjs/plugin/timezone';
+import utc from 'dayjs/plugin/utc';
 import fs from 'fs';
 import Handlebars from 'handlebars';
 import juice from 'juice';
 import Polyglot from 'node-polyglot';
-import { firstValueFrom } from 'rxjs';
 import path from 'path';
-import dayjs from 'dayjs';
-import utc from 'dayjs/plugin/utc';
-import tz from 'dayjs/plugin/timezone';
-import advanced from 'dayjs/plugin/advancedFormat';
-import * as aws from '@aws-sdk/client-sesv2';
-import { TranslationService } from './translation.service';
-import { JurisdictionService } from './jurisdiction.service';
+import { firstValueFrom } from 'rxjs';
+import { ApplicationCreate } from '../dtos/applications/application-create.dto';
 import { Jurisdiction } from '../dtos/jurisdictions/jurisdiction.dto';
-import {
-  LanguagesEnum,
-  ListingEventsTypeEnum,
-  ReviewOrderTypeEnum,
-} from '@prisma/client';
-import { IdDTO } from '../dtos/shared/id.dto';
 import { Listing } from '../dtos/listings/listing.dto';
-import { ApplicationCreate } from '../dtos/applications/application-create.dto';
-import { User } from '../dtos/users/user.dto';
+import { IdDTO } from '../dtos/shared/id.dto';
 import Unit from '../dtos/units/unit.dto';
+import { User } from '../dtos/users/user.dto';
 import { getPublicEmailURL } from '../utilities/get-public-email-url';
-
-dayjs.extend(utc);
-dayjs.extend(tz);
```

Comment:
question: is there a reason these lines were deleted? We added these to add additional options for dayjs